test/probes/isUnsafeImport.spec.js

```diff
@@ -0,0 +1,46 @@
+// Import Node.js dependencies
+import { test } from "node:test";
+import assert from "node:assert";
+
+// Import Internal Dependencies
+import { parseScript, getSastAnalysis } from "../utils/index.js";
+import isUnsafeImport from "../../src/probes/isUnsafeImport.js";
+import isUnsafeCallee from "../../src/probes/isUnsafeCallee.js";
+
+// CONSTANTS
+const kWarningUnsafeImport = "unsafe-import";
+const kWarningUnsafeStmt = "unsafe-stmt";
+test("should detect unsafe import", () => {
+  const str = "const stream = eval('require')('stream');";
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeImport)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeImport);
+  assert.equal(result.kind, kWarningUnsafeImport);
+  assert.equal(result.value, "stream");
+});
+
+test("should detect unsafe statement", () => {
+  const str = "const stream = eval('require')('stream');";
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeCallee)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeStmt);
+  assert.equal(result.kind, kWarningUnsafeStmt);
+  assert.equal(result.value, "eval");
+});
+
+test("should not detect unsafe import", () => {
+  const str = "const stream = eval('fastify')('express');";
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeImport)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeImport);
+  assert.equal(result, undefined);
+});
```

Comment:
I think asserting the number of warnings in each tests could be a bit safer

Reply:
Agree, it's fixed! Some questions below: 

- shall we move the unsafe-stmt test somewhere else or it's good here ?
- i saw you added a folder for tests related to issues, shall we move it there? Or we can consider it as probe tests?

Reply:
Nop that's ok (issues could be useful if there is no other places).